src/instruments/src/RMP/Components/RadioPanelDisplay.tsx

```diff
@@ -23,12 +25,25 @@ const formatFrequency = (frequency: number): string => (frequency / 1000000).toF
 export function RadioPanelDisplay(props: Props) {
     const [lightsTest] = useSimVar('L:A32NX_OVHD_INTLT_ANN', 'Boolean', 1000);
 
-    // If the passed value prop is a number, we'll use formatFrequency to get string format.
-    const value = typeof props.value === 'number' ? formatFrequency(props.value) : props.value;
+    // // If the passed value prop is a number, we'll use formatFrequency to get string format.
+    let value = textDataModeVHF3;
+    let offsetx = '100%';
+
+    if (typeof props.value === 'number') {
+        if (props.value !== 0) {
+            value = formatFrequency(props.value);
+        }
+    } else {
+        value = props.value;
+    }
+
+    if (props.value === 0) {
+        offsetx = '85%';
+    }
```

Comment:
This entire component feels a bit messy now. I expect one bug for example to be that if you're in data mode AND enable lights test, offsetx may be incorrectly at 85%.

How about something more like this:

```typescript
interface Props {
    vale: string | number;
    dataMode?: boolean;
}
```

```typescript
if (lightsTest) {
  content =  (
    <text x={offsetx} y="52%">
      8.8.8.8.8.8
    </text>
  );
}
else if (props.dataMode) {
  content =  (
    <text x="85%" y="52%">
     DATA
    </text>
  );
}
else {
  content =  (
    <text x="100%" y="52%">
     {value}
    </text>
  );
}
return  (
  <svg className="rmp-svg">
    {content}
  </svg>
);
```

Reply:
I'm gonna ask Shomas if he can check the behavior in the real airplane